src/locale/it.json

```
@@ -1243,6 +1243,7 @@
         "channelCode": "Codice canale",
         "noChannelsOption": "Nessun canale trovato",
         "brokerCode": "Codice intermediario",
+        "pspName": "Nome intermediario",
```

se salviamo il nome del PSP 
```suggestion
        "pspName": "Nome PSP",
```